src/Commands/config/ConfigCommands.php

```diff
@@ -170,6 +171,11 @@ public function set($config_name, $key, $value, $options = ['input-format' => 's
             $data = [];
         }
 
+        // Special handling for empty array.
+        if (strtolower($data) == 'null') {
+            $data = NULL;
```

Comment:
Apparently different coding style than Drupal! We need this:
```suggestion
            $data = null;
```